crates/autopilot/src/database/competition.rs

```diff
@@ -18,7 +18,7 @@ use {
 #[derive(Clone, Default, Debug)]
 pub struct Competition {
     pub auction_id: AuctionId,
-    pub reference_scores: HashMap<eth::Address, Score>,
+    pub reference_scores: HashMap<Address, Score>,
```

Comment:
IMO it still makes sense to have the `Address` as a domain type so I would suggest to re-export the type from the `eth` module instead of pulling most basic types from `eth` and only `Address` from `alloy` directly.